css/style.css

```diff
@@ -454,6 +454,11 @@ in the narrow layouts without JS. */
     text-decoration: underline;
 }
 
+.subnav .hide-from-nav
```

Comment:
There was no need to add a new css rule when `.subnav .subnav-duplicate` does the same a few lines above.
-> #1770